client/layout/index.jsx

```diff
@@ -50,6 +50,10 @@ import { handleScroll } from './utils';
 // goofy import for environment badge, which is SSR'd
 import 'calypso/components/environment-badge/style.scss';
 
+// Hotfix for card and button styles hierarchy after <GdprBanner /> removal
+import '@automattic/components/src/button/style.scss';
+import '@automattic/components/src/card/style.scss';
```

Comment:
Uh. I wonder if we can pick a better place for those, now that they've grown to be more than just 1.

Reply:
I think it's the best place, actually. It cannot be anywhere inside async-loaded components (and everything lower in the hierarchy than <Layout /> is async loaded. 

And secondly, I don't want to hide it as I don't think it should become our long-term solution (as described in [post mortem](pbNhbs-4xL-p2)). After being done with privacy work (that has a legal deadline), I think it will be worth exploring how we can get rid of these imports altogether.

Reply:
As long as we have a TODO comment describing we should fix it, I don't see it as a blocker 👍

Reply:
Thanks! I've improved the commentary to include TODO with links to important context